### PR TITLE
Add cname vercel records - puyallup/phs

### PR DESCRIPTION
**Pull request by alexdevz**
so sorry for all the prs!! 😬(x2) but this is for vercel

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1329,6 +1332,9 @@ psn:
   type: CNAME
```
Comment (sampoder) [inline hackclub.com.yaml:1332]: You'll have to remove this one to make it work!

Comment (alexdevz) [reply hackclub.com.yaml:1332]: where would it go then? i need this for the site to work w/ vercel

Comment (sampoder) [reply hackclub.com.yaml:1332]: these are the netlify ones aren't they?

Comment (alexdevz) [reply hackclub.com.yaml:1332]: no these are for vercel?

File: hackclub.com.yaml [hunk 2]
```diff
@@ -1329,6 +1332,9 @@ psn:
...
   value: psnhackclub.netlify.app.
```
Comment (sampoder) [inline hackclub.com.yaml:1333]: they're linked to netlify, the new ones are for vercel

Comment (alexdevz) [reply hackclub.com.yaml:1333]: these aren't for psn, it's for puyallup and phs

Comment (sampoder) [reply hackclub.com.yaml:1333]: ah sorry, my bad!

Comment (alexdevz) [reply hackclub.com.yaml:1333]: It's alright! do you have any idea for my stuff though?

File: hackclub.com.yaml [hunk 3]
```diff
@@ -1245,6 +1245,9 @@ philanthropy:
   type: CNAME
   value: cname.vercel-dns.com.
 phs:
+  - ttl: 1
+    type: CNAME
```
Comment (sampoder) [inline hackclub.com.yaml:1249]:
```suggestion
    type: ALIAS
```
try this

Comment (sampoder) [reply hackclub.com.yaml:1249]: (i know Vercel doesn't say to do this, but it's worked for us in the past)

Comment (alexdevz) [reply hackclub.com.yaml]: alright thanks, let us give it a try!

File: hackclub.com.yaml [hunk 4]
```diff
@@ -1329,6 +1332,9 @@ psn:
   type: CNAME
   value: psnhackclub.netlify.app.
 puyallup:
+  - ttl: 1
+    type: CNAME
```
Comment (sampoder) [inline hackclub.com.yaml:1336]:
```suggestion
    type: ALIAS
```

File: hackclub.com.yaml (hunk 3 above)
Comment (alexdevz) [reply hackclub.com.yaml]: rip didn't work

**Comment by sampoder** [comment]
give this 5min to deploy, then check vercel & let me know if it works

**Comment by alexdevz** [comment]
it works perfect, thank you! sorry for bothering you so much today.

**Pull request merged**